api/scpca_portal/metadata_file.py

```diff
@@ -76,6 +76,25 @@ def load_library_metadata(metadata_file_path: Path):
         return transform_keys(json.load(raw_file), LIBRARY_METADATA_KEYS)
 
 
+def transform_metadata_dict(dict) -> Dict:
```

Comment:
Call this argument `metadata_dict` and add the type hint

Reply:
See commit at fa899d8092b18f0d64284f5fef20819e15fe0e75.